grpc-server/template_test.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/tinkerbell/tink/db/mock"
-	pb "github.com/tinkerbell/tink/protos/template"
+	tb "github.com/tinkerbell/tink/protos/template"
```

Comment:
If you don't rename the package referer, some changes in the file would go away.

Reply:
Yeah, I thought that earlier, but since we are using `pb` as a package referrer for `workflow` and `tb` for package referrer for `template` at many other places so if I use `pb` for `template` package referrer it can cause confusions around the code readers.

Reply:
I have replaced `tb` with `pb` since there was only template package in that file.